### ISSUE TITLE
linter: `unicorn/prefer-default-parameters` misses logical assignments

### ISSUE THREAD

**Issue by lukpsaxo** [oxc-project/oxc#26596]
### What version of Oxlint are you using?

1.82.0

### What command did you run?

oxlint

### What does your `.oxlintrc.json` (or `oxlint.config.ts`) config file look like?

```json
{
    "plugins": ["unicorn"],
    "rules": {
        "unicorn/prefer-default-parameters": "error"
    }
}
```

### What happened?

Oxlint 1.82.0 does not report `??=` or `||=` assignments that can be replaced by a default
parameter. `eslint-plugin-unicorn` 74.0.0 reports both forms as
`unicorn/prefer-default-parameters`.

```js
function example(value) {
    value ??= false;
    return value;
}
```

## Expected

One `unicorn/prefer-default-parameters` diagnostic, equivalent to:

> Prefer default parameters over reassignment for 'value'.

The suggested parameter is `value = false`.

## Actual

No `unicorn/prefer-default-parameters` diagnostic.

Oxlint does report the expanded form `value = value ?? false`, so the difference is specific to
logical assignment syntax.
